common-npm-packages/utility-common/package.json

```diff
@@ -20,6 +20,8 @@
         "@types/node": "^16.11.39",
         "azure-pipelines-task-lib": "^4.11.0",
         "azure-pipelines-tool-lib": "^2.0.7",
+        "azure-pipelines-tasks-azure-arm-rest": "^3.242.0",
```

Comment:
I don't like the fact that we introduce the dependency from one common library to another. I suggest to move this functionality to azure-arm-rest library, since migrated calls are partially already there - https://github.com/search?q=repo%3Amicrosoft%2Fazure-pipelines-tasks-common-packages%20createOidcToken&type=code

Reply:
It's my mistake. I used it in previous iteration of my implementation but forgot to delete the dependency.
It is not needed here